server/src/ClientToDBAPI/ContactedPeople/mainRoute.ts

```diff
@@ -57,15 +57,26 @@ ContactedPeopleRoute.get('/contactStatuses', (request: Request, response: Respon
         investigation: response.locals.epidemiologynumber
     });
     graphqlRequest(GET_ALL_CONTACT_STATUSES, response.locals).then((result: any) => {
-        logger.info({
-            service: Service.SERVER,
-            severity: Severity.LOW,
-            workflow: 'Getting contact statuses',
-            step: 'got respond from the DB',
-            user: response.locals.user.id,
-            investigation: response.locals.epidemiologynumber
-        });
-        response.send(result?.data?.allContactStatuses?.nodes);
+        if (result?.data?.allContactStatuses?.nodes) {
+            logger.info({
+                service: Service.SERVER,
+                severity: Severity.LOW,
+                workflow: 'Getting contact statuses',
+                step: 'got respond from the DB',
+                user: response.locals.user.id,
+                investigation: response.locals.epidemiologynumber
+            });
+            response.send(result?.data?.allContactStatuses?.nodes);
+        } else {
+            logger.error({
+                service: Service.SERVER,
+                severity: Severity.HIGH,
+                workflow: 'Getting contact statuses',
+                step: `got error from the graphql API ${JSON.stringify(result)}`,
```

Comment:
i think error message is in result.errors

Reply:
לפעמים הוא נופל עם error ולא מגיע לcatch אבל שם את השגיאה שם

Reply:
![image](https://user-images.githubusercontent.com/42100027/97879275-74c69200-1d28-11eb-9470-b745a3b6a216.png)

Reply:
https://sachee.medium.com/200-ok-error-handling-in-graphql-7ec869aec9bc